pylti/common.py

```diff
@@ -4,12 +4,19 @@
 """
 
 from __future__ import absolute_import
+
 import logging
 
 import oauth2
-import oauth.oauth as oauth
 from xml.etree import ElementTree as etree
 
+from oauth2 import STRING_TYPES
+try:  # Python 2
+    from urlparse import urlparse
+    from urllib import urlencode
+except ImportError:  # Python 3
+    from urllib.parse import urlparse, urlencode
```

Comment:
I think we should use the [six library](https://pythonhosted.org/six/) for these imports